exercises/pov/canonical-data.json

```diff
@@ -0,0 +1,348 @@
+{
+  "from_pov": {
+    "description": [
+      "Reroot a tree so that its root is the specified node.",
+      "In this way, the tree is presented from the point of view of the specified node.",
+      "The input trees used here are those in the `trees` section of this file.",
+      "",
+      "If appropriate for your track, you may test that the input tree is not modified.",
+      "",
+      "Note that when rerooting upon a target node that has both parents and children,",
+      "it does not matter whether the former parent comes before or after the former children.",
+      "Please account for this when checking correctness of the resulting trees.",
+      "One suggested method is to only check two things:",
+      "1) The root of the returned tree is the root that was passed in to from_pov.",
+      "2) The sorted edge list of the returned tree is the same as the sorted edge list of the expected tree."
+    ],
+    "cases": [
+      {
+        "tree": "singleton",
```

Comment:
potentially contentious point: No `"description":` for any of these. I think I would rather have descriptions.